pyjobshop/ProblemData.py

```diff
@@ -536,6 +536,26 @@ class SetupTime(IterableMixin):
     duration: int
 
 
+@dataclass
+class ModeDependency(IterableMixin):
```

Comment:
Could you also add `ModeDependency` to the following test? `test_Model.py` -> `test_model_to_data()`